admin_manual/appliance/Office.rst

```diff
@@ -13,6 +13,10 @@ Here is an overview of the process:
 
 .. warning::  Access with **HTTPS** using **domain name** is required.
 
+If you encounter an error when trying to open documents, please restart the collabora docker container or simply the appliance.
```

Comment:
Best to write this as [an RST tip admonition](http://docutils.sourceforge.net/docs/ref/rst/directives.html#tip). 

.. tip:: 
  If you encounter an error when trying to open documents, please restart the Collabora Docker container or the appliance.